skynet/modules/stt/vox/connection_manager.py

```diff
@@ -0,0 +1,43 @@
+from typing import List
+
+from fastapi import WebSocketDisconnect
+
+from skynet.logs import get_logger
+from skynet.modules.stt.streaming_whisper.connection_manager import ConnectionManager as BaseConnectionManager
+from skynet.modules.stt.streaming_whisper.utils.utils import TranscriptionResponse
+
+log = get_logger(__name__)
+
+
+class ConnectionManager(BaseConnectionManager):
+    async def process(self, session_id: str, chunk: bytes, timestamp: int, tag: str):
```

Comment:
This is called chunk but we don't call it per chunk, but with a buffer, so let's call it `buffer` or `data` to avoid confusion.